libc/test/src/time/ctime_r_test.cpp

```diff
@@ -0,0 +1,64 @@
+//===-- Unittests for ctime_r ---------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/errno/libc_errno.h"
+#include "src/time/ctime_r.h"
+#include "src/time/time_utils.h"
+#include "test/UnitTest/Test.h"
+#include "test/src/time/TmHelper.h"
+
+using LIBC_NAMESPACE::time_utils::TimeConstants;
+
+// make libc.test.src.time.ctime_r_test
+static inline char *call_ctime_r(time_t *t, char *buffer) {
```

Comment:
why does this function exist? It seems like it would be simpler to just call `LIBC_NAMESPACE::ctime_r` in the tests below.

Reply:
I have now removed the helper functions in both tests.